src/Propel/Generator/Model/Stringifier/TablesStringifier.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license MIT License
+ */
+
+namespace Propel\Generator\Model\Stringifier;
+
+/**
+ * A class for representing a collection of Tables as a String
+ */
+class TablesStringifier
```

Comment:
I don't there is a big difference between TablesStringifier and TableStringifier, but spent 3mins understanding why "stringify" works with an array :)

If the difference is that small please squash them in a single class with 2 public methods.

Reply:
One class has the job of representing many tables as a string, the other class has the job of representing one table as a string. They're similar but not the same. For example, the database __toString method is still messy as it is aware of how to stringify a schema instead of calling a schema stringifier, also the TableStringifer is messy because it mixes all tables concepts inside too.

The code quality score on this project will go back down when we make one class gain another reason to change.

Reply:
Can class docblocks be more clear in describing? Would that help?